includes/class-wc-payments-gateway-wcpay-settings.php

```diff
@@ -0,0 +1,572 @@
+<?php
+/**
+ * Class WC_Payments_Gateway_WCPay_Settings.
+ *
+ * @package WooCommerce\Payments
+ */
+
+use WCPay\Logger;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Gateway settings class for WooCommerce Payments.
+ */
+class WC_Payments_Gateway_WCPay_Settings {
+
+	const OPTIONS = [
```

Comment:
Let's name this settings, as options are still something slightly different.

Reply:
Fixed by 58e0a75b6c0f63f485ade4841c4fb4d40185016a